src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -174,6 +174,14 @@ public class CodeStyleOptions
                 EditorConfigStorageLocation.ForBoolCodeStyleOption("dotnet_style_prefer_is_null_check_over_reference_equality_method"),
                 new RoamingProfileStorageLocation($"TextEditor.%LANGUAGE%.Specific.{nameof(PreferIsNullCheckOverReferenceEqualityMethod)}") });
 
+        internal static readonly PerLanguageOption<CodeStyleOption<bool>> PreferIsNullCheckOverCastAndEqualityOperator = new PerLanguageOption<CodeStyleOption<bool>>(
```

Comment:
since this is only supported in C#, I assume it should be a C#-only option instead (and be in `CSharpCodeStyleOptions`)?